server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -76,6 +79,7 @@ export class TenantManager {
             // if the tenant doesn't have key2, it will be empty string
             // we should fail token generated with empty string as key
             if (!tenantKeys.key2) {
+                Lumberjack.error(`Error with tenant key 1 token verification.`, lumberProperties, error1);
```

Comment:
Maybe we can update the log message to explicitly say that key2 was not present? Something like:
`Error with tenant key 1 token verification while key 2 is not present.`